packages/pipeline/src/scripts/pull_missing_blocks.ts

```diff
@@ -53,8 +53,21 @@ async function getAllMissingBlocks(web3Source: Web3Source): Promise<void> {
 
 async function getMissingBlockNumbers(fromBlock: number): Promise<number[]> {
     console.log(`Checking for missing blocks starting at ${fromBlock}...`);
+    // Note(albrow): The easiest way to get all the blocks we need is to
```

Comment:
Worth noting that with the current size of our tables, this query takes 295ms for me locally. Seems totally fine for now.